frontend/src/app/components/search-form/search-form.component.ts

```diff
@@ -23,9 +23,9 @@ export class SearchFormComponent implements OnInit {
   searchForm: FormGroup;
   @Output() searchTriggered = new EventEmitter();
 
-  regexAddress = /^([a-km-zA-HJ-NP-Z1-9]{26,35}|[a-km-zA-HJ-NP-Z1-9]{80}|[bB]?[a-z]{2,5}1[ac-hj-np-z02-9]{8,87})$/;
-  regexBlockhash = /^[0]{8}[a-fA-F0-9]{56}$/;
-  regexTransaction = /^[a-fA-F0-9]{64}$/;
+  regexAddress = /([a-km-zA-HJ-NP-Z1-9]{26,35}|[a-km-zA-HJ-NP-Z1-9]{80}|[bB]?[a-z]{2,5}1[ac-hj-np-z02-9]{8,87})/;
+  regexBlockhash = /[0]{8}[a-fA-F0-9]{56}/;
```

Comment:
@softsimon Liquid block hashes do not follow this pattern, so they end up being treated as transactions instead (see `bfed8e58a220f6459920a4d7ffc1e822df619a0191068e195845aa237972337e`).

I'd refactor the code below to be more context aware, maybe extracting the logic to a function called `parseNetworkRegex(network, searchText)`.